src/components/bulk-email-tool/BulkEmailTool.jsx

```diff
@@ -10,30 +12,49 @@ import { getCourseHomeCourseMetadata } from './api';
 export default function BulkEmailTool() {
   const { courseId } = useParams();
 
-  const [courseTabData, setCourseTabData] = useState([]);
+  const [courseMetadata, setCourseMetadata] = useState();
 
   useEffect(() => {
     async function fetchTabData() {
       const data = await getCourseHomeCourseMetadata(courseId);
-      const { tabs } = data;
-      setCourseTabData([...tabs]);
+      const { tabs, is_staff: isStaff } = data;
```

Comment:
Would this cover a user who has the `instructor` role? Or `isStaff` dedicated solely to site operators/admin?

Reply:
`isStaff` in this case is given out to people who are on the "staff" list found in studio for the course run. afaik there's no "instructor" role, just admin and staff (this boolean covers both). The original permissions used by the instructor dashboard are slightly different: https://github.com/edx/edx-platform/blob/7f5b4a6a47bfc663e3f980dc6ca2d519feab6253/lms/djangoapps/instructor/permissions.py#L57

Reply:
OK! Sounds good to me. Thanks for looking into this!